cbbpoll/models.py

```diff
@@ -353,3 +353,10 @@ def __repr__(self):
 
     def __str__(self):
         return self.text
+
+
+class Conferences(db.Model):
```

Comment:
Looks like the convention elsewhere is to have singular class and table names... I don't have strong feelings one way or the other, but we might as well stay consistent.